From 1c7bb6beb0dc262a3a379523dbe23611a4a96b48 Mon Sep 17 00:00:00 2001
From: "Jasper St. Pierre" <jstpierre@mecheye.net>
Date: Tue, 23 Jun 2015 14:37:41 -0700
Subject: [PATCH 1/3] setup-shell: Make sure that the shell launches first

On systems where the shell might be slow to start up, we need to make
sure gnome-initial-setup launches after the WM has time to initialize.

Copy these lines from gnome-shell's desktop file to get our setup shell
spawned at the right phase.
---
 data/setup-shell.desktop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/setup-shell.desktop b/data/setup-shell.desktop
index 7770652..7fa1e7f 100644
--- a/data/setup-shell.desktop
+++ b/data/setup-shell.desktop
@@ -3,3 +3,6 @@ Type=Application
 Name=GNOME Shell
 Exec=gnome-shell --mode=initial-setup
 X-GNOME-AutoRestart=true
+X-GNOME-Autostart-Phase=WindowManager
+X-GNOME-Provides=panel;windowmanager;
+X-GNOME-Autostart-Notify=true
-- 
2.4.5

